idea-plugin/src/main/java/com/palantir/gradle/jdks/GradleJdksProjectService.java

```diff
@@ -72,13 +83,22 @@ private ConsoleView initConsoleView() {
             ContentFactory contentFactory = ContentFactory.getInstance();
             Content content = contentFactory.createContent(newConsoleView.getComponent(), "", false);
             toolWindow.getContentManager().addContent(content);
-            // TODO(crogoz): Focus only when error or takes a long time?
-            toolWindow.activate(null);
+            Disposer.register(project, newConsoleView);
```

Comment:
maybe this fixes: https://github.com/palantir/gradle-jdks/issues/394 I wasn't able to repro the issue

Reply:
🙏